countdownTimer.js

```diff
@@ -1,39 +1,80 @@
-// Set the date we're counting down to
-var countDownDate = new Date("Mar 15, 2022 15:37:25").getTime();
+let timer = "5:00"
+document.getElementById("textfield").innerHTML = timer;
 
-// Update the count down every 1 second
-var x = setInterval(function() {
+var x = false;
+function countDown () {
+ 
+  let currenttime = document.getElementsByTagName('p')[0].textContent;
+  let countdowntime = currenttime.split(":");
```

Comment:
This is more of a feedback regarding the architecture:

So it seems like you're hard coding the initial timer string, then you're reading the HTML each second to determine what to do next.

While I'm sure this works for now, have you thought about having the initial timer string, updating it each second, then updating the HTML to reflect the timer string (which in JS lingo is the state of your application)?